issue_template.md

```diff
@@ -0,0 +1,29 @@
+### **Description**
+
+_Please describe the issue you are facing or a feature you would like to be added to the SDK._
+
+### **Repro Steps**
+
+_Please list the steps used to reproduce your issue._
+
+1.
+2.
+
+### **Details**
+
+1. Which .NET SDKs are you using?
```

Comment:
Can we combine 1 and 2 to say - what is your app OS and platform?
[ ] UWP
[ ] Xamarin.Android
[ ] Xamarin.iOS
[ ] Xamarin.Forms